i18n/ru.json

```diff
@@ -97,25 +97,25 @@
     "Add comment":
         "Добавить комментарий",
     "Optional nickname…":
-        "Опциональный никнейм…",
+        "Опциональный никнейм...",
```

Comment:
Please revert this change. Using the "correct" character for three dots (`…`, rendered: …) is intended.